### ISSUE TITLE
Too many flash plugins - false positive

### ISSUE THREAD

**Issue by helix84** [tiliado/nuvolaplayer#159]
**This issue has been fixed in the [master](https://github.com/tiliado/nuvolaplayer) & [3.0.x](https://github.com/tiliado/nuvolaplayer/tree/3.0.x) GIT branches. Thanks for making Nuvola Player better by filing bug reports.**

At first, this fix will be available with the next rebuilt of packages of the [Nuvola Player 3.1 Rolling Releases](https://tiliado.github.io/nuvolaplayer/documentation/3.1/install.html). Later, the fix will be also included in the next bug-fix release of the [Nuvola Player 3.0 Stable Releases](https://tiliado.github.io/nuvolaplayer/documentation/3.0/install.html). To get informed when Nuvola Player 3.0.5 Stable Release is released, follow Nuvola Player on [Facebook](https://www.facebook.com/nuvolaplayer), [Google+](https://plus.google.com/110794636546911932554) or [Twitter](https://twitter.com/NuvolaPlayer); or subscribe to the [Nuvola Player Newsletter](http://eepurl.com/bLbtM1).

---

The same Flash plugin (same path) has been listed twice and detected as duplicate:

![image](https://cloud.githubusercontent.com/assets/201026/9716616/c49cdac6-556b-11e5-98b4-b676677e9e8c.png)

Debian Jessie
nuvolaplayer3 (3.0.0~201508311518.b1fe746-0+1~jessie) amd64

**Comment by helix84** [comment]
Curiously, this happened only after the first startup after installation. Now when I go to "Format support", I only see one entry for flas plugin.

**Comment by fenryxo** [comment]
Hello. That seems to be a glitch in plugin detection code in WebKitGTK. I'm going to postpone this issue to Nuvola Player 3.1 unless it happens again. Let me know in that case.

**Comment by wedens** [comment]
I have the same warning on Arch.

**Comment by fenryxo** [comment]
Helo, @wedens. Are the paths of duplicate flash plugins also same?

**Comment by wedens** [comment]
@fenryxo yeah. same paths, same versions

**Comment by fenryxo** [comment]
In progress...

**Comment by fenryxo** [comment]
**This issue has been fixed in the [master](https://github.com/tiliado/nuvolaplayer) & [3.0.x](https://github.com/tiliado/nuvolaplayer/tree/3.0.x) GIT branches. Thanks for making Nuvola Player better by filing bug reports.**

At first, this fix will be available with the next rebuilt of packages of the [Nuvola Player 3.1 Rolling Releases](https://tiliado.github.io/nuvolaplayer/documentation/3.1/install.html). Later, the fix will be also included in the next bug-fix release of the [Nuvola Player 3.0 Stable Releases](https://tiliado.github.io/nuvolaplayer/documentation/3.0/install.html). To get informed when Nuvola Player 3.0.5 Stable Release is released, follow Nuvola Player on [Facebook](https://www.facebook.com/nuvolaplayer), [Google+](https://plus.google.com/110794636546911932554) or [Twitter](https://twitter.com/NuvolaPlayer); or subscribe to the [Nuvola Player Newsletter](http://eepurl.com/bLbtM1).

**Comment by fenryxo** [comment]
[Nuvola Player 3.0.5 and Diorite 0.2.2 has been released.](https://medium.com/nuvola-news/nuvola-player-3-0-5-and-diorite-0-2-2-released-c170dfb53130)
